Week2/homework/maartjes-work.js

```diff
@@ -47,15 +47,16 @@ const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const hours = tasks.map(task => task.duration / 60);
+  return hours.filter(hour => hour >= 2).reduce((sum, hour) => sum + hour * hourlyRate, 0);
 }
 
 // eslint-disable-next-line no-unused-vars
 const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
 
 // add code to convert `earnings` to a string rounded to two decimals (euro cents)
 
-console.log(`Maartje has earned €${'replace this string with the earnings rounded to euro cents'}`);
+console.log(`Maartje has earned €${earnings.toLocaleString()}`);
```

Comment:
The number on my computer is printed with 3 decimals. This does not meet the requirement of the assignment.